docs/theme.config.tsx

```diff
@@ -0,0 +1,76 @@
+const themeConfig = {
+  head: (
+    <>
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <meta property="og:title" content="Ledger Libs Documentation" />
+      <meta property="og:description" content="Ledger Libs Official Documentation" />
+      <link rel="icon" href="/assets/img/favicon.ico" />
+    </>
+  ),
+  logo: (
+    <div
+      style={{
+        display: "flex",
+      }}
+    >
+      <svg
+        style={{ fill: "currentColor" }}
+        viewBox="73 -10 1 150"
+        width="30"
+        height="27"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <title>Ledger logo</title>
+        <path d="M0 91.6548V128H55.293V119.94H8.05631V91.6548H0ZM138.944 91.6548V119.94H91.707V127.998H147V91.6548H138.944ZM55.3733 36.3452V91.6529H91.707V84.3842H63.4296V36.3452H55.3733ZM0 0V36.3452H8.05631V8.05844H55.293V0H0ZM91.707 0V8.05844H138.944V36.3452H147V0H91.707Z" />
+      </svg>
+      <span style={{ marginLeft: "5px" }}>Ledger Libs</span>
+    </div>
+  ),
+  /**
+   * FIXME:
+   * To update to https://github.com/LedgerHQ/ledger-live/tree/main/docs once (or just before) the PR is merged
+   */
+  docsRepositoryBase:
+    "https://github.com/LedgerHQ/ledger-live/tree/feat/live-0000-documentation/docs",
```

Comment:
Or we could use `develop` as base branch, but since this will be used by the prod / user facing version of the doc, it makes more sense to target main directly (because there is no need for a staging for the doc IMHO)